components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,125 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumbs extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				--d2l-breadcrumbs-background-color: rgb(251, 252, 252);
+				display: block;
+				position: relative;
+			}
+
+			:host::after {
+				background: -webkit-gradient(linear, left top, right top, from(rgba(255, 255, 255, 0)), to(var(--d2l-breadcrumbs-background-color)));
+				background: -webkit-linear-gradient(left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: -o-linear-gradient(left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: linear-gradient(to right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				bottom: 0;
+				content: '';
+				pointer-events: none;
+				position: absolute;
+				right: 0;
+				top: 0;
+				width: 10px;
+			}
+
+			:host-context([dir="rtl"])::after {
+				background: -webkit-gradient(linear, right top, left top, from(rgba(255, 255, 255, 0)), to(var(--d2l-breadcrumbs-background-color)));
+				background: -webkit-linear-gradient(right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: -o-linear-gradient(right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: linear-gradient(to left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				left: 0;
+				right: auto;
+			}
+
+			:host(:dir(rtl))::after {
+				background: -webkit-gradient(linear, right top, left top, from(rgba(255, 255, 255, 0)), to(var(--d2l-breadcrumbs-background-color)));
+				background: -webkit-linear-gradient(right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: -o-linear-gradient(right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: linear-gradient(to left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				left: 0;
+				right: auto;
+			}
```

Comment:
In the future you could write this like this so you don't end up duplicating the rule:
```
:host-context([dir="rtl"])::after,
:host(:dir(rtl))::after {
    ...
}
```

That being said, we should be okay with just single `:host([dir="rtl"])::after { ... }` rule.

Reply:
Yep - `host-context` is not something we can rely on (lots of history behind it, and may never be supported as needed)

Reply:
Done ✔️